### PR TITLE
Create a header and a section

### PR DESCRIPTION
**Pull request by hiromataba**
# Create the header and the section for the mobile version
#Stylying 
#Positioning with flexbox 

#Thanks for taking the time to review my project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@ <h1 id="home-section-title">I am Hiro Glad to See You!</h1>
             contact me.
           </p>
           <div class="home-section-contact">
-            <p>LET'S CONNECT</p>
+            <p><a href="#">LET'S CONNECT</a></p>
             <div class="social-icons">
               <i class="fa fa-twitter"></i>
               <i class="fa fa-linkedin"></i>
```
Comment (descholar-ceo) [inline index.html]: - [ ] The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)

**Comment by hiromataba** [comment]
I accidentally merged my project then I created a new pull request but I really learned from my mistake

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @hiromataba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [x] Make sure that [this commented code](https://github.com/hiromataba/Personal-Portfollio/blob/c022d7f7926809cbd5c0510da14dd08f1652e601/index.html#L20-L30) is removed from your codes, leaving commented code on your codebase it makes code hard to read and hard to debug
- [x] [This hambarger icon](https://github.com/hiromataba/Personal-Portfollio/blob/c022d7f7926809cbd5c0510da14dd08f1652e601/index.html#L15-L19) should be in nav container instead of div container
- [ ] The section uses the font and colors (background and font) defined for the template, kindly check again the template [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- [ ] All changes required on this project should be on this feature branch, not on the main branch, I have noticed that the descriptive readme file is committed on the main branch instead of this feature branch, that is not recommended, any changes related to the project should be on a feature branch before the pull request is approved and merged on the default branch, 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by hiromataba** [comment]
I had merged by accident for the first time that is why that README file
was modified on main branch but I will fix it hanks for the notice so do I
have have to clean the main branch and only work on the feature branch?

**Comment by hiromataba** [comment]
I accidentally merged my project then I created a new pull request that is why the README was modified on the main branch but thanks to your reviews I cannot let that happen again

File: index.html [hunk 2]
```diff
@@ -6,28 +6,25 @@
...
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;400&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <header>
+      <div class="logo"><h1 id="mylogo">My Logo</h1></div>
```
Comment (descholar-ceo) [inline index.html:21]: - [ ] Kindly make this logo a link, normally logo on a website links to the index page

**Review by descholar-ceo — APPROVED** [review]
Hi @hiromataba , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] Just read my comment about your logo in navbar

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**